tools/storage/src/bin/storage-tool.sh

```diff
@@ -6,4 +6,4 @@ for a in "$@"; do
 done
 
 export MARATHON_ARGS
-exec amm-2.12 --predef lib/predef.sc --predef-code "help"
+exec amm-2.13 --predef lib/predef.sc --predef-code "help"
```

Comment:
How does this relate to `tools/storage/bin/amm`?

Reply:
Hmm, a reasonable point. This script is used inside of the docker image but honestly we should switch everything to use the bootstrap script.

The amm docker image should just come with the ~/.ammonite cache directory preloaded.